lib/cocina/models/descriptive_admin_metadata.rb

```diff
@@ -5,8 +5,10 @@ module Models
     class DescriptiveAdminMetadata < Struct
       attribute :contributor, Types::Strict::Array.of(Contributor).meta(omittable: true)
       attribute :event, Types::Strict::Array.of(Event).meta(omittable: true)
-      attribute :language, Types::Strict::Array.of(DescriptiveValue).meta(omittable: true)
+      attribute :language, Types::Strict::Array.of(Language).meta(omittable: true)
```

Comment:
not backwards compatible - be prepared for problematic data?

Reply:
we aren't using this property yet, so probably NBD.